src/plugins/vis_type_vega/public/vega_type.ts

```diff
@@ -43,10 +43,17 @@ export const createVegaTypeDefinition = (
     title: 'Vega',
     getInfoMessage,
     description: i18n.translate('visTypeVega.type.vegaDescription', {
-      defaultMessage: 'Create custom visualizations using Vega and Vega-Lite',
+      defaultMessage: 'Use Vega to create new types of visualizations.',
       description: 'Vega and Vega-Lite are product names and should not be translated',
     }),
+    note: i18n.translate('visTypeVega.type.vegaNote', {
+      defaultMessage: 'Requires programming knowledge.',
+    }),
     icon: 'visVega',
+    group: VisGroups.PROMOTED,
+    groupTitle: i18n.translate('visTypeVega.type.vegaGroupTitle', {
+      defaultMessage: 'Custom visualizations',
```

Comment:
I'd suggest using the singular here "Custom visualization", since we did neither pluralize the other types in this dialog and you'd also creating just a single one when clicking on this button.

Reply:
Agreed